serverless.yml

```diff
@@ -0,0 +1,69 @@
+# Welcome to serverless. Read the docs
+# https://serverless.com/framework/docs/
+
+# Serverless.yml is the configuration the CLI
+# uses to deploy your code to your provider of choice
+
+# The `service` block is the name of the service
+service: hathor-explorer-service
+
+# The `provider` block defines where your service will be deployed
+provider:
+  name: aws
+  runtime: python3.8
+  profile: serverless
+  region: us-east-1
+  vpc:
+    securityGroupIds:
+      - sg-096e4b4e
+    subnetIds:
+      - subnet-06585a4c
+      - subnet-26255a7a
+      - subnet-2ecfbc49
+
+plugins:
+  - serverless-offline
+
+package:
+  individually: true
+
+custom:
+  serverless-offline:
+    httpPort: 3001
+
+functions:
+  get_network_handler:
+    handler: handlers/get_network_handler.handle
+    package:
+      patterns:
+        - '!*'
+        - '!**/*'
+        - 'handlers/get_network_handler.py'
```

Comment:
This doesn't seem right. This file needs many other files to work and it is being included by itself in the final package.
I just ran `serverless package` locally to confirm.

Reply:
Yes, we have to discuss this for deploy. Also put the deps into a lambda layer.

Reply:
Fixed that. Now it will merge patterns with global `package` configuration. Also i moved the exclusions and added the project folders